tests/integration.rs

```diff
@@ -7771,11 +7771,21 @@ mod live_migration {
         _test_live_migration(false, false, false, true, false)
     }
 
+    #[test]
+    fn test_live_migration_watchdog_local() {
```

Comment:
Nit: why didn't you add this directly in the commit covering `virtio-watchdog`?

Reply:
PR updated.

I wasn't sure whether we want to include the "local" variation of the tests for watchdog (and balloon) mainly due to the long testing time for watchdog (normally took over 3 mins on my dev machine). Looks like we are fine with it for now. We can always remove them later if we want.